src/tools/wasm-as.cpp

```diff
@@ -51,6 +53,12 @@ int main(int argc, const char *argv[]) {
       .add("--debuginfo", "-g", "Emit names section and debug info",
            Options::Arguments::Zero,
            [&](Options *o, const std::string &arguments) { debugInfo = true; })
+      .add("--source-map", "-sm", "Emit source map to the specified file",
+           Options::Arguments::One,
+           [&sourceMapFilename](Options *o, const std::string &argument) { sourceMapFilename = argument; })
+      .add("--source-map-url", "-su", "Use specified string as source map URL",
+           Options::Arguments::One,
+           [&sourceMapUrl](Options *o, const std::string &argument) { sourceMapUrl = argument; })
```

Comment:
these are duplicated in two files. i am currently getting rid of most of the existing duplication in #1023. it would be good to do the same thing for these options, however, i understand if you want to land this first and leave the refactoring to followup work.